src/app/Shared/Model/Client_Isolate.model.ts

```diff
@@ -55,14 +55,14 @@ export type DbCollection = (Record<DbKeyWithStringValue, string> & {
 
 export const DbKeyCollection: DbKey[] = [

```

Comment:
If this array defines the filters in the app and not the keys in the DB, i suggest to rename it to something like:
MainFilterEntries or MainFilterList.